src/scala/com/github/johnynek/bazel_deps/Resolver.scala

```diff
@@ -63,7 +63,11 @@ class Resolver(servers: List[MavenServer], resolverCachePath: Path) {
     val ex = ml.dependencies.excludes(m.unversioned)
     val exclusions = new util.ArrayList[Exclusion]()
     for (elem <- ex){
-      val exclusion = new Exclusion(elem.group.asString, elem.artifact.asString, "", "jar")
+      val exclusion = new Exclusion(
+        elem.group.asString,
+        elem.artifact.getArtifact,
+        elem.artifact.getClassifier.orNull,
+        elem.artifact.getPackaging)
```

Comment:
Example of exclusions including `packaging`:

``
  com.amazonaws:
    DynamoDBLocal:
      lang: java
      version: "1.11.86"
      exclude:
        - "com.almworks.sqlite4java:sqlite4java-win32-x86:dll"
        - "com.almworks.sqlite4java:sqlite4java-win32-x64:dll"
        - "com.almworks.sqlite4java:libsqlite4java-osx:dylib"
        - "com.almworks.sqlite4java:libsqlite4java-linux-i386:so"
        - "com.almworks.sqlite4java:libsqlite4java-linux-amd64:so"
```

I suspect, but haven't tried that you could also exclude `"com.almworks.sqlite4java:libsqlite4java-linux-i386:*"` (asterisk at the end) to exclude _all_ packaging types for that dependency (if more than one were present). Seems to be a feature of aether.